### ISSUE TITLE
Changeable image value for aerospike-init

### ISSUE THREAD

**Issue by metbog** [aerospike/aerospike-kubernetes-operator#123]
Hello,

We get the following error:
`14m         Warning   Failed                   pod/aerospike-cluster-0-0                             Failed to pull image "aerospike/aerospike-kubernetes-init:0.0.15": rpc error: code = Unknown desc = Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit`

We want to use our proxy for docker images. We try to replace the value by:
```
      podSpec:
        initContainers:
          - name: aerospike-init
            image: proxy/aerospike/aerospike-kubernetes-init:0.0.15
```
but get the error:
`Error: admission webhook "vaerospikecluster.kb.io" denied the request: cannot use reserved container name: aerospike-init`

Is it possible to control?

**Comment by sud82** [comment]
Hi @metbog, Currently we do not support custom docker image for the aerospike-init container.
https://github.com/aerospike/aerospike-kubernetes-operator/issues/5

But, this is in our roadmap.

**Comment by sud82** [comment]
Released in 2.1.
https://github.com/aerospike/aerospike-kubernetes-operator/releases/tag/2.1.0
